Vagrantfile

```diff
@@ -25,33 +25,34 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   # Example for VirtualBox:
   #
 
-  # Set number of CPUs to use for 
+  # Set number of CPUs to use for compilation
```

Comment:
& actually this is untrue, as we are for some reason adding 1 to num_cpus on line 45! :D